### ISSUE TITLE
marxspawn

### ISSUE THREAD

**Issue by marxspawn** [cop1000/cop1000#99]
Paste your web site's published URL here:  ~~https://marxspawn.github.io/Ray-Anderson.github.io/~~
## https://marxspawn.github.io/

Check each of the following boxes to ensure you can score :100:

- Bootstrap is used. 
- Web site contains images.
- Web site contains text.
- Web site contains external links.
- All of the web site's content can be accessed just by scrolling.
- Users can navigate to different sections of the webpage by clicking buttons in the navigation.
- The web site contains original content.
- The web site doesn't contain inappropriate content.
- The web site is published to GitHub Pages.
- The link to the web site is submitted in the class repository.

**Comment by marxspawn** [comment]
Well... Its not finished, but i did cover everything on the list :metal: :neckbeard: 

- Turned in ~~on~~ ~~time~~

**Comment by hectorsector** [comment]
@marxspawn it's a start but I couldn't verify this criteria was met:

> Web site contains images.
> Users can navigate to different sections of the webpage by clicking buttons in the navigation.
> The web site contains original content.

Make sure that your images aren't broken links as they currently are. You'll also have to replace the content that comes with the template with your own original content.

Also, if you change the repository name to `marxspawn.github.io` you'll notice that you can visit your web site at https://marxspawn.github.io/

**Comment by marxspawn** [comment]
@hectorsector 

Changes
- changed repi name to https://marxspawn.github.io
- added picture
- link to top of page clearly visible

I'm still not 100%... But, I did learn a lot about containers and elements. I used one of the github templates and have been playing around with the stylesheet changing variables (basically figuring out how they work) which is why I think it has taken me so long. I am not a big fan of apps doing all the work for me. I could have easily just plugged a bunch of photos with text and been done. But I want to  understand what's going on behind the scenes more.

Thank you for this project, 
I have tried using github for a while now and was discouraged because I didn't understand the whole branch, pull, merge concept. Here in the last week I have gotten a better grasp on the whole concept. I'm not a pro yet, but am definitely more comfortable with it.
